.github/workflows/publish.sh

```diff
@@ -4,21 +4,28 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# from https://cloud.google.com/sdk/docs/downloads-apt-get
```

Comment:
Happy to see this go, but how is this going to work now. I must be missing something

Reply:
gcloud is already on the machine